.github/workflows/release.yml

```diff
@@ -12,17 +12,17 @@ jobs:
       - name: Prepare repository
         run: git fetch --unshallow --tags
 
-      - name: Use Node.js 14.x
+      - name: Use Node.js 16.x
         uses: actions/setup-node@v1
         with:
-          node-version: 14.x
+          node-version: 16.x
 
       - name: Install dependencies
-        uses: bahmutov/npm-install@v1
+        uses: yarn install --ignore-scripts
```

Comment:
This breaks the CI, it should be `run: yarn install --ignore-scripts` instead of `uses: yarn install --ignore-scripts`